.github/workflows/std.yml

```diff
@@ -195,8 +182,8 @@ jobs:
       - name: Configure K8S Cluster Access
         shell: bash
         run: |
-          echo "Assuming role '$(aws sts get-caller-identity)' in cluster 'lace-prod-eu-central-1'."
```

Comment:
is this an intentional change?

Reply:
New cluster identifier, it is intentional. Maybe I am missing something?